ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -93,18 +94,18 @@ namespace Ikarus {
      * @param p_neumannBoundaryLoad The Neumann boundary load function (optional, default is LoadDefault).
      */
     template <typename VolumeLoad = utils::LoadDefault, typename NeumannBoundaryLoad = utils::LoadDefault>
-    KirchhoffLoveShell(const Basis& globalBasis, const typename LocalView::Element& element, double emod, double nu,
+    KirchhoffLoveShell(const Basis &globalBasis, const typename LocalView::Element &element, double emod, double nu,
```

Comment:
Instead of passing `emod` and `nu`, can we change the constructor such that it accepts `const Material& p_mat` similar to `nonlinearelastic.hh`? It is a bit more general then? Or do you think it should be in a different PR because this modification is needed for `LinearElastic` as well.

Reply:
The problem here is that the material has to be stress-resultant based and not stress-based. Therefore, the current material does not implement such a material